File: Npc/11004161.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11004161: Nurse Nora
/// </summary>
public class _11004161 : NpcScript {
    protected override int First() {
        return 10;
    }

    // Select 0:
    // $script:0730132107010547$
    // - Are you here to see the doctor? Do you have an appointment?
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (10, 0):
                // $script:0730132107010548$
                // - Sigh, no time to rest. Too many patients to get to.
                return -1;
        }

        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (10, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11004161.py
""" 11004161: Nurse Nora """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 10

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0730132107010547$
        # - Are you here to see the doctor? Do you have an appointment?
        return None # TODO

    def __10(self, pick: int) -> int:
        # $script:0730132107010548$
        # - Sigh, no time to rest. Too many patients to get to.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (10, 0):
            return Option.CLOSE
        return Option.NONE
